chemtsv2/parallel_mcts.py

```diff
@@ -44,12 +44,15 @@ def is_high_priority(self, tag):
         return tag >= self.PRIORITY_BORDER.value
 
 
-class Tree_Node():
-    def __init__(self, state, parentNode=None, reward_calculator=None, conf=None):
-        # todo: these should be in a numpy array
+class MPNode():
+    def __init__(self, position=['&'], parentNode=None, reward_calculator=None, conf=None):
+        # todo: the payload of MPI should be in a numpy array. consider using @property for implementation
         # MPI payload [node.state, node.reward, node.wins, node.visits, node.num_thread_visited, node.path_ucb]
-        self.state = state
-        self.childNodes = []
+        if position is None:
```

Comment:
I'm wondering if there is a case where `position` can be `None`. Just want to ensure that this check is necessary.